web/themes/custom/vote_gov/src/sass/uswds-overrides/_override-usa-header.scss

```diff
@@ -27,6 +27,16 @@
   margin-top: 0;
 }
 
+.usa-header {
+  [role=search] {
+    @include at-media($theme-header-min-width) {
+      float: right;
+      max-width: 340px;
+      width: 100%;
+    }
```

Comment:
Because USWDS doesn't offer up a variable for the size we need we can replace this with the standard media query that you used in the other file:
```
@media (min-width: 40em) {
      max-width: 340px;
    }
```

We also can inherit the float and width values. So this is all you should need.